internal/k8s/controllers/service_controller_reload.go

```diff
@@ -71,7 +71,11 @@ func (r *ServiceReconciler) reprocessAll(ctx context.Context, req ctrl.Request)
 	for _, service := range services.Items {
 		service := service // so we can use &service
 		if filterByLoadBalancerClass(&service, r.LoadBalancerClass) {
-			level.Debug(r.Logger).Log("controller", "ServiceReconciler", "filtered service", req.NamespacedName)
+			level.Debug(r.Logger).Log("controller", "ServiceReconciler", "filtered service", service.Name, "reason", "not type LoadBalance")
```

Comment:
When doing this first pass where we filter the services without ip, we must return errRetry so the reconciliation loop will be triggered again